.circleci/config.yml

```diff
@@ -18,8 +18,66 @@ jobs:
           PATH=$HOME/miniconda3/envs/pandas-dev/bin:$HOME/miniconda3/condabin:$PATH
           LD_PRELOAD=$HOME/miniconda3/envs/pandas-dev/lib/libgomp.so.1:$LD_PRELOAD
           ci/run_tests.sh
+  build-aarch64:
+    parameters:
+      cibw-build:
+        type: string
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.large
+    environment:
+      TAG = << pipeline.git.tag >>
+      TRIGGER_SOURCE = << pipeline.trigger_source >>
+    steps:
+      - checkout
+      - run:
+          name: Check if build is necessary
+          command: |
+            # Check if tag is defined or TRIGGER_SOURCE is scheduled
+            if [[ -n ${TAG} ]]; then
+                export IS_PUSH="true"
+            elif [[ $TRIGGER_SOURCE == "scheduled_pipeline" ]]; then
+                export IS_SCHEDULE_DISPATCH="true"
+            # Look for the build label
+            elif ! (curl https://api.github.com/repos/pandas-dev/pandas/issues/$CIRCLE_PR_NUMBER | jq '.labels' | grep -q 'Build'); then

```

Comment:
Is it worth putting a retry on the `curl`?

Reply:
I added specifying "wheel build" in the commit message as an alternative.

There's not much we can do if we get rate limited.